index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Tribute page</title>
+</head>
+<body>
+    <main id="main">
+
+        <h1 id="title">Eliud Kipchoge </h1>
+        <div id="img-div">
+            <img id='image' src="https://ih1.redbubble.net/image.1716595426.7382/flat,128x,075,f-pad,128x128,f8f8f8.jpg" alt="Image not loaded">
+            <figcaption id="img-caption">Kipchoge running the NYC marathon</figcaption>
+        </div>
+        <div id="tribute-info">
+            <p>Eliud Kipchoge (born 5 November 1984) is a Kenyan professional long-distance runner who competes in the marathon and formerly competed at the 5000 metre distance. He is the 2016 and 2020 Olympic marathon winner. In addition, he set the world record in the marathon with a time of 2:01:39 at the 2018 Berlin Marathon. His run broke the previous world record by 1 minute and 18 seconds.[2] He is widely regarded as the greatest marathon runner and one of the greatest sportsmen in history.</p>
+            <a id="tribute-link" href="" target="_blank">Tribute</a>

```

Comment:
When using an `<a>`  tag, always be sure to fill in the `href` attribute. This will prevent you to encounter linter errors 😁